### PR TITLE
ci(travis): re-enable build against node 12

### PR DESCRIPTION
**Pull request by DRoet**
## Description

Pretty sure fibers was causing the issues with node 12, hence the bump in version. 
@KaelWD might know more

## Motivation and Context

(yarn + yarn build) + ci was broken in node.js 12

## Markup:

<details>

```vue
// Paste your FULL Playground.vue here
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by DRoet** [comment]
travis seems happy, I'm not sure about the now deployment...

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: node_js
-node_js: '11.14.0'
+node_js: 'node'
```
Comment (KaelWD) [inline .travis.yml]:
```suggestion
node_js: '12'
```

Se we don't have this all over again when 13 is released

**Comment by DRoet** [comment]
> Need to change the node version in now too.

is this suppose to be somewhere in the `now.json` file?

**Comment by KaelWD** [comment]
I think it might be in package.json

**Comment by DRoet** [comment]
Is there any way to see the `now` deployment logs? I can't really tell why it is failing.

**Comment by KaelWD** [comment]
Oh yeah I guess that's private haha.
image

**Comment by DRoet** [comment]
@KaelWD I think its should be good now

**Pull request merged**